app/assets/javascripts/bastion/components/bst-table.directive.js

```diff
@@ -17,7 +19,22 @@ angular.module('Bastion.components')
                 'rowSelect': '@',
                 'rowChoice': '@'
             },
-            controller: 'BstTableController'
+            controller: 'BstTableController',
+            link: function (scope) {
+                var resize = function () {
+                    angular.element($window).trigger('resize');
+                };
+
+                // Trigger resize after resource $promise is resolved
+                scope.$watch('table.resource', function (resource) {
+                    if (resource && resource.hasOwnProperty('$promise')) {
+                        resource.$promise.then(resize);
+                    }
+                });
+
+                // Trigger resize when rows change
+                scope.$watch('table.rows', resize);
```

Comment:
Yeah, the rows changing is for when a new row is added, the resource resolving is for when the resource is resolved.  I didn't have much luck just using the watch on `table.rows` because I think it gets called before the browser renders the scrollbar.